build/scripts/build.ps1

```diff
@@ -29,6 +29,7 @@ param (
     [switch]$packAll = $false,
     [switch]$binaryLog = $false,
     [switch]$deployExtensions = $false,
+    [switch]$launch = $false,
```

Comment:
Consider adding an error to `Process-Arguments` when `$build` and `$launch` are specified but not `$deploy`.

Reply:
The alternative is to make `-build -launch` imply `-deployExtensions`. The advantage is the following can be used at that point:

```
.\build.cmd -release -launch
```

I can go either way. Let me know which change, if any, you prefer.

Reply:
I actually liked the idea of `-launch` being completely independent. In the case of `-build -launch` it seems most likely the developer isn't doing what they think they're doing.

Reply:
➡️ Validation is now added